poetry/installation/authenticator.py

```diff
@@ -138,6 +152,29 @@ def _get_credentials_for_netloc_from_config(
     ) -> Tuple[Optional[str], Optional[str]]:
         credentials = (None, None)
 
+        for (repository_name, repository_netloc) in self._get_repository_netlocs():
+            if netloc == repository_netloc:
+                auth = self._password_manager.get_http_auth(repository_name)
+
+                if auth is None:
+                    continue
+
+                credentials = (auth["username"], auth["password"])
+                break
+
+        return credentials
+
+    def get_certs_for_url(self, url: str) -> Dict[str, pathlib.PosixPath]:
```

Comment:
```suggestion
    def get_certs_for_url(self, url: str) -> Dict[str, pathlib.Path]:
```

Think the same change was made for `_get_certs_for_netloc_from_config()`

Reply:
Applied your suggestions. Thank you!